website/sidebars.js

```diff
@@ -13,10 +13,31 @@ module.exports = {
       "advanced/user-object",
       "advanced/public-api",
       "advanced/caching",
-      "advanced/team-collaboration",
+      {
+        "User Management": [
```

Comment:
Should be `Team Management` as in the Dashboard

Reply:
Sidebar item ordering by importance:
- Team Management
  - Team Management Basics
  - Auto-assign users
  - SAML
  - Domain validation

Reply:
✔️